src/factories/interfaces/IImmutable_PIM_Factory_v1.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+// Internal Interfaces
+import {IOrchestrator_v1} from
+    "src/orchestrator/interfaces/IOrchestrator_v1.sol";
+import {IOrchestratorFactory_v1} from
+    "src/factories/interfaces/IOrchestratorFactory_v1.sol";
+import {IBondingCurveBase_v1} from
+    "@fm/bondingCurve/interfaces/IBondingCurveBase_v1.sol";
+
+// Internal Dependencies
+import {ERC20Issuance_v1} from "src/external/token/ERC20Issuance_v1.sol";
+
+/**
+ * @title   Immutable PIM Factory
```

Comment:
Our usual naming includes the word "Inverter" in the `@title` now, so "Inverter Payment Router" instead of just "Payment Router". I think we should be consistent here as well then?

"Immutable Inverter PIM Factory" or so? And for Restricted as well.